### PR TITLE
Refs[TASK][34791]Create UI meal detail

### PR DESCRIPTION
**Pull request by GiangGiang1105**
## Related Tickets
- [#Task 34791: Create UI meal detail](https://edu-redmine.sun-asterisk.vn/issues/34791)
## WHAT
- Create UI meal detail
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/activity_meal_detail.xml [hunk 1]
```diff
@@ -1,7 +1,96 @@
...
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="@dimen/dp_10"
+                android:layout_weight="8"
+                android:fontFamily="@font/redressed"
+                android:paddingEnd="@dimen/dp_10"
+                android:textColor="@color/color_08BECA"
+                android:textSize="@dimen/dp_30" />
+
+            <androidx.appcompat.widget.AppCompatImageButton
+                android:id="@+id/btnFavourite"
```
Comment (huytq-2422) [inline app/src/main/res/layout/activity_meal_detail.xml]:
```suggestion
                android:id="@+id/buttonFavourite"
```

File: app/src/main/res/layout/activity_meal_detail.xml [hunk 2]
```diff
@@ -1,7 +1,96 @@
...
+            android:weightSum="10"
+            android:layout_marginTop="@dimen/dp_20"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/viewVideo">
+
+            <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/textMeal"
+                android:layout_width="wrap_content"
```
Comment (huytq-2422) [inline app/src/main/res/layout/activity_meal_detail.xml]: Tại sao phải nhét vào 1 Linear Layout để nó thêm phân khúc ViewGroup lồng nhau thế? 
Có thể trải phẳng ra constraint layout

File: app/src/main/res/layout/activity_meal_detail.xml [hunk 3]
```diff
@@ -1,7 +1,92 @@
...
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/textMeal"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dp_20"
+            android:fontFamily="@font/redressed"
+            android:paddingEnd="@dimen/dp_10"
+            android:textColor="@color/color_08BECA"
+            android:textSize="@dimen/dp_30"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_meal_detail.xml]: sp

File: app/src/main/res/layout/activity_meal_detail.xml [hunk 4]
```diff
@@ -1,7 +1,92 @@
...
+            app:layout_constraintEnd_toStartOf="@+id/buttonFavourite"
+            app:layout_constraintHorizontal_bias="0.033"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/viewVideo" />
+
+        <androidx.appcompat.widget.AppCompatImageButton
+            android:id="@+id/buttonFavourite"
+            android:layout_width="48dp"
+            android:layout_height="34dp"
+            android:layout_gravity="center_vertical"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_meal_detail.xml]: hard code

File: app/src/main/res/layout/activity_meal_detail.xml [hunk 5]
```diff
@@ -1,7 +1,92 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/dp_20"
+            android:layout_marginTop="@dimen/dp_20"
+            android:layout_marginRight="@dimen/dp_20"
+            android:background="@color/color_A1F4FA"
+            android:fontFamily="@font/redressed"
+            android:padding="@dimen/dp_10"
+            android:textColor="@color/color_08257E"
+            android:textSize="@dimen/dp_20"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_meal_detail.xml]: sp

File: app/src/main/res/layout/activity_meal_detail.xml [hunk 6]
```diff
@@ -1,7 +1,92 @@
...
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_meal_detail.xml]: sp

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**